pulsar-broker/src/main/java/com/yahoo/pulsar/broker/service/ServerCnx.java

```diff
@@ -300,7 +392,7 @@ protected void handleSubscribe(final CommandSubscribe subscribe) {
                         }) //
                         .exceptionally(exception -> {
                             log.warn("[{}][{}][{}] Failed to create consumer: {}", remoteAddress, topicName,
-                                    subscriptionName, exception.getCause().getMessage());
+                                    subscriptionName, exception);
```

Comment:
this will be `CompletionException` and it will not be helpful, we have to log `exception.getCause()`